doc/attributes.md

```diff
@@ -15,6 +15,7 @@ Requires order of attributes. When more attributes are in one `#[]`, eg. `#[One,
 Sniff provides the following settings:
 
 * `order`: required order of attributes. Supports prefixes, eg. `ORM\`.
+* `orderAlphabetically`: order attributes alphabetically. Boolean value, default `false`.
```

Comment:
We should say that only one of these options can be set.

Reply:
> I want to have the ability to sort by custom order and alphabetically simultaneously

It does not make sense to me and I won't merge it. I'm sorry.

Reply:
I am sad to hear this, it is a very useful feature.

Reply:
Example use case. Let's imagine we have the following attributes:
```php
#[ORM\Index(columns: ['w_column'])]
#[ORM\Index(columns: ['a_column'])]
#[ORM\Index(columns: ['b_column'])]
#[Assert\ZAssert]
#[Assert\AAssert]
#[Assert\BAssert]
```

It will be easier to read and search if attributes are sorted alphabetically and in the same time by custom order:
- `ORM\`
- `Assert\`

Like this:
```php
#[ORM\Index(columns: ['a_column'])]
#[ORM\Index(columns: ['b_column'])]
#[ORM\Index(columns: ['w_column'])]
#[Assert\AAssert]
#[Assert\BAssert]
#[Assert\ZAssert]
```

Reply:
```
#[ORM\Index(columns: ['a_column'])]
#[ORM\Index(columns: ['b_column'])]
#[ORM\Index(columns: ['w_column'])]
```

This will not be fixed/sorted by this sniff. The sniff sorts only names, not content.

And I'm ok with the option, I just don't think it should be combined with the second option.

Reply:
I have just updated it - now, if alphabet sorting is enabled, it will consider content when sorting.

Would you tell me what exactly confuses you about the possibility of simultaneous sorting?

Alphabet sorting is disabled by default. Only those who need it will use it.

Reply:
> Would you tell me what exactly confuses you about the possibility of simultaneous sorting?

I just don't think it's the right thing.

Reply:
@kukulich I have discussed this with the team, and now I agree that sniff must allow only one sorting option.

I have updated the PR.

Would you please check?